mindmeld/components/_config.py

```diff
@@ -522,6 +514,35 @@
     },
 }
 
+DEFAULT_NORMALIZERS = ["NFC"]
+
+DEFAULT_EN_NORMALIZERS = [
+    'RemoveAposAtEndOfPossesiveForm',
+    'RemoveAdjacentAposAndSpace',
+    'RemoveBeginningSpace',
+    'RemoveTrailingSpace',
+    'ReplaceSpacesWithSpace',
+    'ReplaceUnderscoreWithSpace',
+    'SeparateAposS',
+    'ReplacePunctuationAtWordStartWithSpace',
+    'ReplacePunctuationAtWordEndWithSpace',
+    'ReplaceSpecialCharsBetweenLettersAndDigitsWithSpace',
+    'ReplaceSpecialCharsBetweenDigitsAndLettersWithSpace',
+    'ReplaceSpecialCharsBetweenLettersWithSpace',
+    'RemoveSpecialCharsBeforePipe',
+    'ReplaceEndBracketAndFollowingSpecialCharsBeforeSWithSpace',
+    'Lowercase',
+    'ASCIIFold'
+]
```

Comment:
Maybe move these to a ``config_helpers`` file? We can do a refactor of the config file to move a lot of the supporting code later. This file has become quite verbose.

Reply:
Good point, however, after discussing this with Vijay, we decided to use the existing Normalization rules for English for all languages. So we are no longer maintaining defaults for each language